message/validation/validation.go

```diff
@@ -569,13 +580,13 @@ func (mv *messageValidator) consensusState(messageID spectypes.MessageID) *Conse
 		Role:   messageID.GetRoleType(),
 	}
 
-	if _, ok := mv.index.Load(id); !ok {
-		cs := &ConsensusState{
-			Signers: hashmap.New[spectypes.OperatorID, *SignerState](),
-		}
-		mv.index.Store(id, cs)
+	if csItem := mv.state.Get(id); csItem != nil {
```

Comment:
Get extends the time to live of that item, is 2 epochs always enough or we do want to extend the life of an item once touched? if its enough extending can be disabled.

Reply:
2 epochs should be enough because messages after 32+1 slots are considered late. Extending the life might be useful for preventing malicious behavior because the TTL would be updated on any action, so the violator would become closer to the punishment threshold on each violation.